### ISSUE TITLE
SD Maid doesn't seem to work on Android 11.

### ISSUE THREAD

**Issue by osikiri** [d4rken/sdmaid-public#3875]
SD Maid version 4.15.13

Android 11 final beta on Pixel 3 XL

I run CorpseFinder and SystemCleaner

but they always return the result of 0 items to be cleaned.

Permissions are allowed for the app.

**Comment by d4rken** [comment]
Can you provide a log?
https://github.com/d4rken/sdmaid-public/wiki/Reporting-a-bug#debugrun-log

**Comment by osikiri** [comment]
[sdmaid_logfile_1598698031149.txt.zip](https://github.com/d4rken/sdmaid-public/files/5145099/sdmaid_logfile_1598698031149.txt.zip)

I just made a log file and attached it here.

**Comment by d4rken** [comment]
Thanks, will look into it.
There's definitely some fixes I need to do for Android 11, which I already have in a sepperate branch, but I can't upload that to Google Play yet which means I can't merge the fixes yet. If it is related to that, then I'm not sure how to solve it right now.

https://developer.android.com/preview/privacy/storage#change-details

>Due to COVID-19 related considerations, apps that target Android 11 and need the MANAGE_EXTERNAL_STORAGE permission cannot be uploaded to Google Play until early 2021. This includes new apps as well as updates to existing apps. To learn more, read the updated policy from the Policy Help Center.

**Comment by rtromao** [comment]
> > Due to COVID-19 related considerations, apps that target Android 11 and need the MANAGE_EXTERNAL_STORAGE permission cannot be uploaded to Google Play until early 2021. This includes new apps as well as updates to existing apps. To learn more, read the updated policy from the Policy Help Center.

Is that means the Appcleaner won't work until that date? 🤔

**Comment by d4rken** [comment]
> Is that means the Appcleaner won't work until that date?

I think you mean a different issue, maybe https://github.com/d4rken/sdmaid-public/issues/3939

**Comment by rtromao** [comment]
> I think you mean a different issue, maybe #3939

You're right. That what I'm talking about. I can see you plan to fix in the v5. Do you have an ETA for that? This is my must have feature.

**Comment by d4rken** [comment]
Hoping for at least a beta in the next days.

**Comment by spiritETG** [comment]
I just discovered that an outside app named Solid Explorer has a different approach to accessing /Android/Data.  When this popup shows up: 
<img src="https://user-images.githubusercontent.com/3746534/132473621-ab00e179-f16f-4215-9497-ef298e51f64f.png" width="400">

As you press [Grant], it teleports you straight into Android/Data and all you have to do is to give it access...
<img src="https://user-images.githubusercontent.com/3746534/132473899-046f570f-14ff-412b-9eb6-ffec531251b5.png" width="400">
<img src="https://user-images.githubusercontent.com/3746534/132473969-accf269f-8d7a-43af-bb62-d72a7c835948.png" width="400">

Reading and writing permission works as normal (from a consumer perspective), you can still delete the content inside app's data and etc..

The outside app I'm referring to is this if you need a reference: https://play.google.com/store/apps/details?id=pl.solidexplorer2
